packages/twenty-website/src/database/sqlite/migrations/0001_easy_norman_osborn.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE `githubStars` (
+	`id` text PRIMARY KEY NOT NULL,
+	`numberOfStars` integer
```

Comment:
It feels odd to have this structure.
I would rather (1) create a more generic key/value table (e.g githubInfo with last release number,  total star count, etc.). 
Or (2) make this a time series table by introducing a date field (and then you only retrieve the most recent one).